python/tribute.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+from paraview.simple import *
+
+# create a new 'PNG Series Reader'
+tributepng = PNGSeriesReader(FileNames=['tribute.png'])
+
+# create a new 'Calculator'
+calculator1 = Calculator(Input=tributepng)
+calculator1.ResultArrayName = 'originalData'
+calculator1.Function = 'sqrt(PNGImage_X*PNGImage_X+PNGImage_Y*PNGImage_Y)'
+
+# create a new 'Tetrahedralize'
+tetrahedralize1 = Tetrahedralize(Input=calculator1)
+
+# create a new 'TTK PersistenceDiagram'
+tTKPersistenceDiagram1 = TTKPersistenceDiagram(Input=tetrahedralize1)
+tTKPersistenceDiagram1.ScalarField = ['POINTS', 'originalData']
+tTKPersistenceDiagram1.InputOffsetField = [None, '']
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.